src/utils/floating-ui.ts

```diff
@@ -197,8 +199,19 @@ export interface FloatingUIComponent {
    * @param delayed – (internal) when true, it will reposition the component after a delay. the default is false. This is useful for components that have multiple watched properties that schedule repositioning.
    */
   reposition(delayed?: boolean): Promise<void>;
+
+  /**
+   * Specifies the component's layout.
+   *
+   * Possible values: "vertical" or "horizontal".
```

Comment:
Can you add some info regarding this only being needed for floating-ui owners that support arrows? Additionally, can you add a note on this using the `@State` decorator (similar to https://github.com/Esri/calcite-components/blob/be3a8b2414794562f5efa8a8223fdacd93192d1f/src/utils/locale.ts#L210 or https://github.com/Esri/calcite-components/blob/c683f6d4d52ecfffca9ffcb2ab7dc4dc758d0cad/src/utils/t9n.ts#L122)?